vllm/worker/model_runner.py

```diff
@@ -823,14 +862,31 @@ def build(self) -> ModelInputForGPU:
         """
         # Combine and flatten intermediate data.
         input_tokens = []
+        inputs_embeds_: list[torch.Tensor] = []
```

Comment:
```suggestion
        inputs_embeds = list[torch.Tensor]()
```

Maybe also provide the other two lists with better type annotations